docs/usage.rst

```diff
@@ -204,6 +204,31 @@ Now we're going to get the base URL and some image links::
      'image4_thumb.jpg',
      'image5_thumb.jpg']
 
+JMESPath Selector
+-----------------
+Whenever the selected elements possess JSON format, it is possible to use  ``.jmespath()`` method from
+:class:`~parsel.selector.Selector`.
+
+For example, given the selector is parsing the following element::
+
+    >>> text = '{"user":[{"name":"A","age":"25"},{"name":"B","age":"32"},{"name":"C","age":"19"}]}'
+    >>> jmespath_selector = Selector(text=text)
+
+It is possible to extract the age of the users as an integer by using the following line of code::
+
+    >>> jmespath_selector.jmespath("user[*].age.to_number(@)").getall()
+    [25,
+     32,
+     19]
+
+JMESPath selectors can be chained with css and xpath selectors,
+as well as use `.re()` and `re_first()` and the other methods.
```

Comment:
```suggestion
JMESPath selectors can be chained with CSS and XPath selectors,
and they also allow you to use `.re()`, `re_first()`, and the other methods.
```

Reply:
I removed this part of the doc since it creates the idea that they are different. Yes, they are, but we don't need to explicitly mention it in this way. Seems more appropriate to simply present them as selectors and the user will understand they can be chained and all have the same methods.